app/services/schools/register_ect.rb

```diff
@@ -48,6 +49,7 @@ def register!
         update_school_last_choices!
         create_teacher!
         @ect_at_school_period = start_at_school!
+        create_training_period! if lead_provider.present?
```

Comment:
It's not clear what the intent behind the `if` statement is here, it could be made a little more obvious that we only want to create them for `provider-led` training.

Reply:
Good shout - have updated 👍🏽